File: src/main/java/web/practicas/demo/model/dto/FechasDTO.java
package web.practicas.demo.model.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class FechasDTO {
    String limiteinferior;
    String limitesuperior;
    String nombreatributo;
}

File: Front/src/app/Interfaces/dto/FechasDTO.ts
export class FechasDto{
    limiteinferior:string;
    limitesuperior:string;
    nombreatributo:string;

    constructor(limiteinferior:string,limitesuperior:string,nombreatributo){
        this.limiteinferior=limiteinferior;
        this.limitesuperior=limitesuperior;
        this.nombreatributo=nombreatributo;
    }
}